dev/clint/src/clint/rules/track_api_usage_outermost.py

```diff
@@ -0,0 +1,33 @@
+import ast
+from typing import Optional
+
+from clint.resolver import Resolver
+from clint.rules.base import Rule
+
+
+class TrackApiUsageOutermost(Rule):
```

Comment:
```suggestion
class TrackApiUsageTopmost(Rule):
```

Let's rename the rule to match the mssage.